zulipterminal/model.py

```diff
@@ -2071,3 +2071,17 @@ def poll_for_events(self) -> None:
                         self.controller.raise_exception_in_main_thread(
                             sys.exc_info(), critical=False
                         )
+
```

Comment:
You've split commits more here, but note the commit text - you're updating model.py, so 'ui' doesn't apply to the 'area' for the title.